tests/unit/modules/network/sonic/test_sonic_lldp_interfaces.py

```diff
@@ -47,6 +47,25 @@ def tearDown(self):
         self.mock_config_edit_config.stop()
         self.mock_get_interface_naming_mode.stop()
 
+    def validate_config_requests(self):
```

Comment:
What is the reason for this overloaded function? Why isn't the parent class validate_config_requests being directly used?

Reply:
the original function sorts only the request paths. This new function sorts the payload as well. LLDP TLVs are passed in the payload and this leads to UT failure as the payload contains the TLVs in random order that cannot be compared.

Reply:
Okay. Thanks for explaining.

Resolved.